velox/functions/lib/DateTimeFormatter.cpp

```diff
@@ -189,28 +190,40 @@ constexpr int monthsFullLength[] = {7, 8, 5, 5, 3, 4, 4, 6, 9, 7, 8, 8};
 /// \param padFront if the padding is in front of the content or back of the
 /// content.
 template <typename T>
-std::string padContent(
+void padContent(
     const T& content,
-    char padding,
-    size_t totalDigits,
-    bool padFront = true) {
+    const char padding,
+    const size_t totalDigits,
+    common::OutputByteStream& outStream,
```

Comment:
Why can't we just pass char* and use std::fill?

Reply:
Just changed to use std::fill/memcpy and remove the use of `OutputByteStream` in this patch. Thanks!